static/app/views/performance/landing/widgets/utils.tsx

```diff
@@ -29,15 +29,21 @@ function setWidgetStorageObject(localObject: Record<string, string>) {
 
 const mepQueryParamBase = {};
 
-export function getMEPQueryParams(mepContext: MetricsEnhancedSettingContext) {
+export function getMEPQueryParams(
+  mepContext: MetricsEnhancedSettingContext,
+  forceAuto?: boolean
+) {
   let queryParams = {};
   const base = mepQueryParamBase;
-  if (mepContext.shouldQueryProvideMEPAutoParams) {
+  if (mepContext.shouldQueryProvideMEPAutoParams || forceAuto) {
     queryParams = {
       ...queryParams,
       ...base,
       dataset: 'metricsEnhanced',
     };
+    if (forceAuto) {
+      return queryParams;
+    }
```

Comment:
This can probably be kept as a switch by re-arranging providers and cleaning them up, but for now early exists with the queryExtras blurb for `metricsEnhanced`